pkg/gatewayserver/gatewayserver.go

```diff
@@ -102,13 +109,14 @@ func New(c *component.Component, conf *Config, opts ...Option) (gs *GatewayServe
 		return nil, err
 	}
 	if len(forward) == 0 {
-		forward[""] = []types.DevAddrPrefix{{}}
+		forward["ttn.lorawan.v3.GsNs:cluster"] = []types.DevAddrPrefix{{}}
```

Comment:
For clarity, leave this to `""` and explicitly use this interface for empty. See https://github.com/TheThingsNetwork/lorawan-stack/pull/1198/files?file-filters%5B%5D=.go#r315555734